docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,44 @@
+# Percona Monitoring and Management 2.43.0
+
+| **Release date** | July , 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+This release introduces this and that
+
+## Release highlights
+
+
+
+
+
+![!image](../_images/.png)
+
+## Improvements
+
+- [PMM-13133](https://perconadev.atlassian.net/browse/PMM-13133) - 
+
+- [PMM-13054](https://perconadev.atlassian.net/browse/PMM-13054) - The default number of log lines for each log file returned by `httsp://<pmm-server>/logs.zip` endpoint has been increased from 1,000 to 50,000. 
+
+Additionally, the endpoint now includes a customizable line-count parameter in the download URL, allowing you to specify a custom number of log lines or opt for unlimited log size. For more information, see the [API documentation](https://percona-pmm.readme.io/reference/logs).
+
+
+## Maintenance
+
+### QAN update: Upgrade to `pg_query_go` v5 and impact on PostgreSQL 9.x
+
+As part of our ongoing maintenance efforts, we have upgraded the `pg_query_go` package to version 5. This latest version helps parse PostgreSQL queries in QAN, provides improved functionality, and enables subsequent upgrades of important dependencies.
+
+However, with this change, QAN will no longer be able to list PostgreSQL database tables for versions 9.4, 9.5, and 9.6. Since all PostgreSQL 9.x versions have already reached their end-of-life status, we recommend upgrading to a supported version of PostgreSQL.
+
+If you're looking to upgrade, you can easily [install the latest version of Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/13/installing.html). 
```

Comment:
I checked to see which distribution is `latest`. Our docs say it's [this one](https://docs.percona.com/postgresql/16/installing.html).